main.js

```diff
@@ -1,11 +1,28 @@
 var $button = document.querySelector('.start')
+var $currentTime = document.querySelector('.timeElapsed')
+var timeElapsed = parseInt($currentTime.textContent, 10)
+var stop = setInterval(updateTime, 1000)
```

Comment:
Instead of starting your interval here, you should only start it when the button is clicked. Try something like this.

```js
var intervalId = null

$button.addEventListener('click', function () {
  if (intervalId === null) {
    intervalId = setInterval(updateTime, 1000)
  }
  else {
    clearInterval(intervalId)
    intervalId = null
  }
})
```